File: ishicodes/solutions/2026-06-16/Q77.py
# Problem: Multiply matrices.

n=int(input()); a=[list(map(int,input().split())) for _ in range(n)]; b=[list(map(int,input().split())) for _ in range(n)]; [print(*(sum(a[i][k]*b[k][j] for k in range(n)) for j in range(n))) for i in range(n)]

File: ishicodes/solutions/2026-06-16/Q77.c
/*
 * Problem: Multiply matrices.
 */

#include<stdio.h>
int main() {
    int n;scanf("%d",&n);int a[n][n],b[n][n],c[n][n];for(int i=0;i<n;i++)for(int j=0;j<n;j++)scanf("%d",&a[i][j]);for(int i=0;i<n;i++)for(int j=0;j<n;j++)scanf("%d",&b[i][j]);for(int i=0;i<n;i++){for(int j=0;j<n;j++){c[i][j]=0;for(int k=0;k<n;k++)c[i][j]+=a[i][k]*b[k][j];printf("%d ",c[i][j]);}printf("\n");}
    return 0;
}

